reddit.go

```diff
@@ -143,6 +143,9 @@ func (c *Reddit) getSubmission(id string) (models.PostListingChild, error) {
 	})
 	ret := &models.PostListing{}
 	json.Unmarshal(ans, ret)
+	if len(ret.GetChildren()) < 1 {
+		return "", fmt.Errorf("id not found")
```

Comment:
Build failed: https://travis-ci.com/thecsw/mira/builds/130496105

```
github.com/thecsw/mira
235# github.com/thecsw/mira
236./reddit.go:147:10: cannot use "" (type string) as type models.PostListingChild in return argument
237./reddit.go:160:10: cannot use "" (type string) as type models.Comment in return argument
```

Change the empty string to `nil`